app/uk/gov/hmrc/customs/managesubscription/audit/Auditable.scala

```diff
@@ -41,4 +46,15 @@ class Auditable @Inject() (auditConnector: AuditConnector, @Named("appName") app
       )
     )
 
+  def sendExtendedDataEvent(
+    transactionName: String,
+    path: String = "N/A",
```

Comment:
Why the default to "N/A"?  You are setting a url in the one place calling this method.

Reply:
I copied this method from frontend service, but it's a good point.

I'll refactor the default arguments in both repositories, we're not using them. Probably `"N/A"` value is a correct one to put when path is missing, but it's better to do it explicitly.

Reply:
Method above in line 38 has the same default parameter